cmd/kpod/history.go

```diff
@@ -23,22 +22,34 @@ const (
 	idTruncLength        = 13
 )
 
-// historyOutputParams stores info about each layer
-type historyOutputParams struct {
-	ID        string `json:"id"`
-	Created   string `json:"created"`
-	CreatedBy string `json:"createdby"`
-	Size      string `json:"size"`
-	Comment   string `json:"comment"`
+// historyTemplateParams stores info about each layer
+type historyTemplateParams struct {
```

Comment:
I really dislike having two nearly identical structs with the only difference being whether the fields are strings or not, but I can't think of a better way of doing this.

Reply:
I agree, but with the new formats package I also couldn't see a way around this.